geospaas/vocabularies/managers.py

```diff
@@ -119,8 +119,8 @@ class PlatformManager(VocabularyManager):
                     short_name='Short_Name',
                     long_name='Long_Name')
 
-    def get_by_natural_key(self, short_name):
-        return self.get(short_name=short_name)
+    #def get_by_natural_key(self, short_name):
```

Comment:
It is better to remove completely rather than comment out. We wills still have it in git history. And the code remains cleaner without.